pkg/artifactory/resource_artifactory_local_npm_repository.go

```diff
@@ -0,0 +1,34 @@
+package artifactory
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+var npmLocalSchema = mergeSchema(baseLocalRepoSchema, map[string]*schema.Schema{})
```

Comment:
Since it's identical to the generic repo, this line is entirely unneeded. Delete it and just use the baseLocalRepoSchema directly

Reply:
My first attempt was to use baseLocalRepoSchema directly ... which fails to save the state in hcl. Will understand this failure and fix it.

Reply:
Used baseLocalRepoSchema directly.